includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -166,6 +166,22 @@ public function get_job_id() {
 		return absint( $this->job_id );
 	}
 
+	/**
+	 * Checks if it is forced application default field
+	 *
+	 * @return bool
+	 */
+	protected function is_forced_application_default_field() {
+		/**
+		 * Force application field to skip email / URL validations and sanitizer
+		 *
+		 * @since 1.34.1
```

Comment:
For now let's set `@since`'s to 1.x.x so we can flag to change it when we release. I moved this to 1.34.2, but the next release might be 1.35.0.

Reply:
Okay! Should I always do like this, using only the major version? Or is it only for this case?